app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FoodsController < ApplicationController
+  def search
+    @foods = FoodFacade.food_search(params[:search])
```

Comment:
The param that's coming through from the form that you inherited with this code uses the key of `:q`, not `:search`. So, essentially nothing is getting passed to the `food_search` method since the parameter of `search` is not present.